discover/reference/articles/04-form-fields.markdown

```diff
@@ -0,0 +1,66 @@
+# Form Field Types [](id=form-field-types)
+
+A form without fields is no form at all. To meet your form-building needs,
+Liferay Forms gives you many useful and highly configurable field types.
+
+![Figure 1: Out of the box form field types.](../images/forms-field-types.png)
+
+Form Text
+: Users do not enter data into Form Text fields. The form creator enters text
+that form users see displayed on the form. This is useful for longer
+instructions.
+
+![Figure 2: A form text field.](../images/forms-form-text.png)
+
+Text Field : Users enter text into these fields. For example, a Full Name field
+will often be a text field. By default, a text field keeps all input on a single
+line of text. To accommodate longer responses, choose the multiline setting when
+configuring the text field. Place limits on the text users can enter (numbers
+from 1-10, or email addresses, for example) by using the Text field's validation
+options.
+
+![Figure 3: A multiline text form.](../images/forms-multiline.png)
+
+Select from List
+: Users select one (or more, if configured to allow it) options from a dropdown
+list of choices. Choices can be entered manually or automatically populated by
+a data provider. For example, a Country of Residence field would probably be a
+select from list field populated by a Countries of the World data provider. 
+
+![Figure 4: A select from list field.](../images/forms-select-list.png)
+
+Single Selection
+: Using a radio button, users select one option from a list of options displayed
+on the form. 
+
+![Figure 5: A single selection field.](../images/forms-single-selection.png)
+
+Date
+: Users select a date using a datepicker. For example, a Birth Date field would
+probably use the Date field type.
+
+![Figure 6: A date field.](../images/forms-date.png)
+
+Multiple Selection/Single Checkbox
+: Users select one or more options from check boxes (or switchers, if
+configured).
+
```

Comment:
Hey @rbohl,

It's fine for me have one section for these both fields, however the Single Checkbox is available only in the DE and not in the CE, so can we find some way to highlight this? I think it would be important. Additionally, the **Out of the box form field types** image doesn't show the Single Checkbox, which makes this note very relevant.

Reply:
How does this look? https://github.com/rbohl/liferay-docs/commit/9a8b4662ebb777bfcbae180463302ebaccd1ebc5

Reply:
Lovely, @rbohl! But I think the use of @product@ can introduce a bug in the documentation when the @product@ is equal to CE. Can you please confirm it?

Reply:
Correct. I've addressed your remaining comments here: https://github.com/rbohl/liferay-docs/commit/6b4d3b75049994dfb57a7078b565ea0b68c27fa9

I'll send this for final review. Thanks @natocesarrego ! If you see anything else don't hesitate to mention it.